packages/react-native-usb/src/index.ts

```diff
@@ -0,0 +1,144 @@
+/* eslint-disable no-console */
+import { NativeModulesProxy, EventEmitter, Subscription } from 'expo-modules-core';
+
+import { ReactNativeUsbModule } from './ReactNativeUsbModule';
+import { NativeDevice, OnConnectEvent, WebUSBDevice } from './ReactNativeUsb.types';
+
+const DEBUG_LOGS = true;
```

Comment:
Should not this be `false` by default?

Reply:
Later yes, but I expect this to be quite useful in first few days/weeks after merge so I plan to change it to false later